editor/property_selector.cpp

```diff
@@ -552,18 +574,21 @@ PropertySelector::PropertySelector() {
 	VBoxContainer *vbc = memnew(VBoxContainer);
 	add_child(vbc);
 	//set_child_rect(vbc);
+
 	search_box = memnew(LineEdit);
 	vbc->add_margin_child(TTR("Search:"), search_box);
 	search_box->connect("text_changed", callable_mp(this, &PropertySelector::_text_changed));
 	search_box->connect("gui_input", callable_mp(this, &PropertySelector::_sbox_input));
 	search_options = memnew(Tree);
+	search_options->set_select_mode(search_options->SELECT_SINGLE);
```

Comment:
Select single as default; this only applies when have made a new PropertySelector not when switching between tracks whilst using the same PropertySelector.